packages/form-js-viewer/src/render/components/form-fields/Datetime.js

```diff
@@ -210,14 +210,17 @@ export function Datetime(props) {
 Datetime.config = {
   type,
   keyed: true,
-  label: 'Date time',
+  name: 'Date time',
   group: 'basic-input',
   emptyValue: null,
   sanitizeValue: sanitizeDateTimePickerValue,
-  create: (options = {}) => {
+  create: (options = {}, isNew) => {
```

Comment:
I just think we should give a more descriptive name

```suggestion
  create: (options = {}, is NewLabel) => {
```

Reply:
I'll rename it all to isNewField (the fact that the label is new is only an implementation detail of the fact that the field is new).